### DISCUSSION TITLE
Releases policies

### DISCUSSION THREAD

**Discussion by victordidenko** [videojs/video.js#8801] [Q&A]
Is there any place about releases policies described?

I checks dependencies updates each week. Today as usual I checked dependencies, and I was surprised that video.js version has been jumped from 8.12.0 up to 8.16.1
```diff
-    "video.js": "^8.12.0",
+    "video.js": "^8.16.1",
```

In releases page I see, that all other intermediate versions are marked as "pre-release":
- 8.12.0 — release
- 8.13.0 — pre-release
- 8.14.0 — pre-release
- 8.14.1 — pre-release
- 8.15.0 — pre-release
- 8.16.0 — pre-release
- 8.16.1 — release
- 8.17.0 — pre-release

What is going on here? Why is that so?

**Comment by amtins** [comment]
@victordidenko yes, please refer to the following section [videojs-releases](https://github.com/videojs/video.js/blob/57c27f8e5ca8ba79340871aa497317b61ec492b6/COLLABORATOR_GUIDE.md#videojs-releases).
